i18n/en.toml

```diff
@@ -30,25 +30,22 @@ other = "No"
 [latest_version]
 other = "latest version."
 
-[shortcode_version_check_mustbe]
+[version_check_mustbe]
 other = "Your Kubernetes server must be version "
 
-[shortcode_version_check_mustbeorlater_start]
-other = "Your Kubernetes server must be version "
-
-[shortcode_version_check_mustbeorlater_end]
-other = " or later."
+[version_check_mustbeorlater]
+other = "Your Kubernetes server must be or later than version "
 
```

Comment:
Hello @seokho-son, looks good. One nit:
The value for `version_check_mustbeorlater` seems a bit unclear to me. Would this string work:
`Your Kubernetes server must be at or later than version `
Here is a page where `version_check_mustbeorlater` renders in English:
https://deploy-preview-12418--kubernetes-io-master-staging.netlify.com/docs/tasks/run-application/run-stateless-application-deployment/

Reply:
@kbhawkey Thank you ! It looks good :+1: I updated PR.